### ISSUE TITLE
half time

### ISSUE THREAD

**Issue by loveuhaxball** [haxball/haxball-issues#97]
can be added otomaatic half-time option ? So score constant 
stay

**Comment by Kozlov98** [comment]
Yes, that is a real good idea, in some official tournaments that is the game mode, two halfs. With swaped sides and shirts.

**Comment by loveuhaxball** [comment]
@basro

**Comment by helppe** [comment]
i agree too.
it will allow both teams to be blue and red.
